.circleci/config.yml

```diff
@@ -43,7 +43,7 @@ lint-template: &lint-template
 jobs:
   lint:
     docker:
-      - image: python:3.12-slim
+      - image: python:3.13-slim
```

Comment:
[Python 3.12 is about to be switched to maintenance mode](https://devguide.python.org/versions), so I figure it's a good time to change the default version to 3.13